rules/windows/defense_evasion_solarwinds_backdoor_service_disabled_via_registry.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/12/14"
+maturity = "production"
+updated_date = "2020/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification of a Service start type to disabled. An adversary may abuse this technique to any relevant
+security services.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Service Disabled via Registry"
+references = [
+    "https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html",
+]
+risk_score = 47
+rule_id = "b9960fef-82c6-4816-befa-44745030e917"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+registry where registry.path : "HKLM\\SYSTEM\\ControlSet*\\Services\\*\\Start" and registry.data.strings == "4" and
+ process.name : (
+     "SolarWinds.BusinessLayerHost*.exe", 
+     "ConfigurationWizard*.exe", 
```

Comment:
IMHO it's the opposite its too specific. since when one disable a service (via Service Control Manager) the process that does the change is services.exe. The process in this rule are all potential target of loading a backdoored DLL.